templates/grpc.gemspec.template

```diff
@@ -31,7 +31,7 @@
     s.require_paths = %w( src/ruby/lib src/ruby/bin src/ruby/pb )
     s.platform      = Gem::Platform::RUBY
 
-    s.add_dependency 'google-protobuf', '~> 3.7'
+    s.add_dependency 'google-protobuf', '~> 3.8.0'
```

Comment:
I believe just `3.8` is correct for ruby. @apolcyn will know more.

Reply:
Done. But I do see 3.8.0 here: https://rubygems.org/gems/google-protobuf, and it seems to work fine. Maybe 3.8 would automatically catch 3.8.0.1 or 3.8.1 release. Would we want to make changes to GRPC when that happen or just let it upgrade automatically?